crawler/src/main/java/com/brandwatch/minibcr/crawler/model/Resource.java

```diff
@@ -0,0 +1,11 @@
+package com.brandwatch.minibcr.crawler.model;
+
+public class Resource {
+    private final String title;
+    private final String text;
+
```

Comment:
Perhaps you'd also want to store the Date/Time of a resource as it will be helpful in the future when you try to fetch mentions for a certain date